packages/salmon-provider-tiingo/package.json

```
@@ -0,0 +1,49 @@
+{
+  "private": false,
+  "name": "@defichain/salmon-provider-tiingo",
+  "version": "0.0.0",
+  "description": "Price provider for the tiingo api",
+  "keywords": [
+    "DeFiChain",
+    "DeFi",
+    "Blockchain",
+    "API",
+    "Bitcoin"
+  ],
+  "repository": "DeFiCh/salmon",
+  "bugs": "https://github.com/DeFiCh/salmon/issues",
+  "license": "MIT",
+  "contributors": [
+    {
+      "name": "DeFiChain Foundation",
+      "email": "engineering@defichain.com",
+      "url": "https://defichain.com/"
+    },
+    {
+      "name": "DeFi Blockchain Contributors"
+    },
+    {
+      "name": "DeFi Salmon Contributors"
+    }
+  ],
+  "main": "dist",
+  "types": "dist",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "tsc"
+  },
+  "peerDependencies": {
+    "bignumber.js": "^9.0.1"
+  },
+  "dependencies": {
+    "@defichain/jellyfish-json": "next",
```

```suggestion
    "@defichain/salmon-price-functions": "0.0.0",
```

mono repo not properly configured?